FitGoal/FitGoal/Utilities/GradientView.swift

```diff
@@ -23,4 +23,16 @@ class GradientView: UIView {
             gradientLayer.colors = colors.map { $0.cgColor }
         }
     }
+    
+    var startPoint: CGPoint = .zero {
+        didSet {
+            gradientLayer.startPoint = startPoint
+        }
+    }
+    
+    var endPoint: CGPoint = .zero {
```

Comment:
The default endpoint shouldn't be zero, it should be the default end point of the gradient layer.